### ISSUE TITLE
MVP & Schema Review

### ISSUE THREAD

**Issue by vawhitehouse** [joshebrani/MashBook#1]
## MVP
- [x] Add estimated timeline for each MVP
- [x] Should users be able to edit their comments?
- [x] We want the Production README to be one of the 7 MVPs that you will be focusing on - I would recommend moving the search MVP to a bonus MVP and only work on that once everything is completed

## Schema
- [x] Facebook has nested commenting so your `comments` table should have an option `parent_comment_id` column
- [ ] You may want to come up with a more distinct way to differentiate between the two user foreign keys in the `friends` table - `requester`/`requestee` maybe?
- [ ] You'll likely also want a `status` column for `friends` since one user has to request the friend and the other user has to accept - you need to keep track of that inbetween time as well as the acceptance or rejection

**Comment by joshebrani** [comment]
Thanks for the feedback Val!

Regarding the nested comments - the app academy website shows that as a bonus so I was thinking that might be a lot to handle. Would you highly recommend I include that?

**Comment by vawhitehouse** [comment]
You are completely correct, I missed that. Nested comments should be a bonus MVP so you won't need the `parent_comment_id` starting out.
